### ISSUE TITLE
Ruby org timeframe_versions API is not consistent

### ISSUE THREAD

**Issue by AlonHalimi** [rubygems/rubygems.org#3855]
Consecutive calls to `https://rubygems.org/api/v1/timeframe_versions.json` with the same parameters don't always give the same output

## Steps to Reproduce
Browse to https://rubygems.org/api/v1/timeframe_versions.json?from=2009-07-24T00:00:00Z&to=2009-07-30T00:00:00Z&page=518 many times and search for `"version":"0.24.0"` (or even `0.24.0`) in the result (for `name`=`gen`, `platform`=`ruby`)
you'll see that sometimes it appears, sometimes it doesn't

## Expected Behavior
Each request to `https://rubygems.org/api/v1/timeframe_versions.json` with a given (`from`, `to`, `page`) parameter tuple should give exactly the same output

## Current Behavior
Consecutive calls to `https://rubygems.org/api/v1/timeframe_versions.json` with a given (`from`, `to`, `page`) parameter tuple don't always give the same output

## Possible Solution
maybe in `version.rb`
```
  def self.created_between(start_time, end_time)
    where(created_at: start_time..end_time).order(:created_at)
  end
```
the `order` should add more fields than`created_at` only? (in case many versions have been created at the same date) 

## Environment
Browser and its version: Chrome - any version - but also `curl` or any other HTTP client
RubyGems version: N/A

## Additional Context
N/A

**Comment by simi** [comment]
Hello @AlonHalimi and thanks for the report, detailed explanation and solution proposal. I have opened fix at #3887. Feel free to take a look.

**Comment by AlonHalimi** [comment]
Hello  @simi 
this looks good :)
thanks a lot !
